fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -338,6 +342,29 @@ class BrowserToolbarCFRPresenter(
                     )
                 }
             },
+            action = {
+                FirefoxTheme {
+                    Text(
+                        text = if (shouldShowOptedInCFR) {
+                            stringResource(id = R.string.review_quality_check_second_cfr_action)
+                        } else {
+                            stringResource(id = R.string.review_quality_check_first_cfr_action)
+                        },
```

Comment:
❓ Can you explain for my understanding what's happening here?

Reply:
Same idea as on line 335, because both shopping CFRs have the same specs, if we are to show the second (opted in CFR), we just need to show the corresponding text.

Reply:
gotcha, thanks for the explanation!